sys/dev/ce/if_ce.c

```diff
@@ -2636,11 +2636,9 @@ MODULE_DEPEND (ce, sppp, 1, 1, 1);
 #endif
 #ifdef KLD_MODULE
 DRIVER_MODULE (cemod, pci, ce_driver, ce_devclass, ce_modevent, NULL);
-MODULE_PNP_INFO("U16:vendor; U16:device;D:#", pci, cemod, ce_devs,
-     sizeof(ce_devs[0]), nitems(ce_devs));
+PCI_PNP_INFO(ce_devs);
```

Comment:
where were ce_devs converted to the new pci_device_table ?

Reply:
It was converted earlier in.. https://github.com/bsdimp/freebsd/commit/991e00e955e1803e1c7e645bf8802351f089cf40